deepsensor/train/train.py

```diff
@@ -18,6 +18,10 @@ def set_gpu_default_device():
             # Set default GPU device
             torch.set_default_device("cuda")
             B.set_global_device("cuda:0")
+        elif torch.backends.mps.is_available():
+            # Set default GPU device
+            torch.set_default_device("mps")
+            B.set_global_device("mps:0")
```

Comment:
I was thinking something more along the lines of `set_default_device("mps")`. I feel like if a user calls `set_gpu_default_device()` they will expect to use a GPU, not an MPS. What do you think?